grouper/models/group_service_accounts.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy import Column, ForeignKey, Index, Integer
+from sqlalchemy.orm import backref, relationship
+
+from grouper.models.base.model_base import Model
+
+
+class GroupServiceAccount(Model):
+    """Service accounts owned by a group.
+
+    A group may own zero or more service accounts. This table holds the mapping between a Group and
+    the ServiceAccount objects it owns.
+    """
+
+    __tablename__ = "group_service_accounts"
+    __table_args__ = (
+        Index(
+            "group_service_account_idx",
+            "group_id", "service_account_pk",
+            unique=True
+        ),
+    )
+
+    id = Column(Integer, primary_key=True)
+
+    group_id = Column(Integer, ForeignKey("groups.id"), nullable=False)
+    group = relationship("Group", backref="service_accounts", foreign_keys=[group_id])
+
+    service_account_pk = Column(Integer, ForeignKey("service_accounts.id"), nullable=False)
```

Comment:
do you have a philosophy in your mind in regards to naming columns `_id` vs `_pk`? it seems we aren't entirely consistent as it is or at least i can't discern the pattern.

Reply:
I was just trying to copy our existing inconsistency. I'd be totally okay with just using `_id` everywhere; I think that would be better than whatever we're currently doing.

Reply:
yeah, i'd agree with using `_id` for new fields.